src/backend.org

```diff
@@ -3176,6 +3179,50 @@ class TestNodalCentre(TestCase):
         self.assertEqual(centre_name, "IIIT-Hyd")
 #+END_SRC
 
+
+***** getCentreStatus: NodalCentre --> CentreStatus
+#+BEGIN_SRC python :tangle src/db.py :eval no 
+    def get_centre_status(self):
+        current_app.logger.debug("get name of the NodalCentre: %s"  % self.centre_status)
+        return self.centre_status
+#+END_SRC
+
+***** Test getCentreStatus: NodalCentre --> CentreStatus 
```

Comment:
Python unit test case defined for the method **getCentreStatus** is wrong. This test case might fail while running. Please update it correctly.